bots/images/scripts/windows-8.bootstrap

```diff
@@ -0,0 +1,66 @@
+
+set -ex
+
+BASE=$(dirname $0)
+
+ORIGIN="$1"
+VM_NAME=win8.1
+
+test -e IE11.Win8.1.For.Windows.VMware.zip || wget https://az412801.vo.msecnd.net/vhd/VMBuild_20141027/VMware/IE11/Windows/IE11.Win8.1.For.Windows.VMware.zip
+test -e selenium.zip || wget http://selenium-release.storage.googleapis.com/2.53/IEDriverServer_Win32_2.53.1.zip -O selenium.zip
+test -e java-installer.exe || wget http://javadl.oracle.com/webapps/download/AutoDL?BundleId=216432 -O java-installer.exe
+test -e selenium.jar || wget https://goo.gl/Lyo36k -O selenium.jar
+
+unzip -o IE11.Win8.1.For.Windows.VMware.zip
+unzip -o selenium.zip
+
+cat <<'EOF' > Autorun.inf
+[autorun]
+open=install.exe
+label=Selenium installer
+EOF
+
+cat <<'EOF' > selenium.bat
+netsh interface ipv4 set address name="Ethernet 3" static 10.111.112.10 255.255.240.0 || netsh interface ipv4 set address name="Ethernet 4" static 10.111.112.10 255.255.240.0
+timeout 20
+START "hub" java -jar C:\selenium\selenium.jar -role hub
+timeout 20
+java -jar C:\selenium\selenium.jar -role node  -hub http://localhost:4444/grid/register -browser browserName="internet explorer"
+timeout 50
+EOF
+
+cat <<'EOF' > install.bat
+net file 1>nul 2>nul && goto :run || powershell -ex unrestricted -Command "Start-Process -Verb RunAs -FilePath '%comspec%' -ArgumentList '/c %~fnx0 %*'"
+goto :eof
+:run
+netsh advfirewall firewall add rule name="Open Port 4444" dir=in action=allow protocol=TCP localport=4444
+netsh advfirewall firewall add rule name="Open Port 5555" dir=in action=allow protocol=TCP localport=5555
+NetSh Advfirewall set allprofiles state off
+d:\java-installer.exe /s
+mkdir c:\selenium
+echo f | xcopy /f /y d:\selenium.jar c:\selenium\selenium.jar
+echo f | xcopy /f /y d:\selenium.bat c:\selenium\selenium.bat
+echo f | xcopy /f /y d:\IEDriverServer.exe c:\selenium\IEDriverServer.exe
+echo f | xcopy /f /y d:\IEDriverServer.exe c:\Windows\system32\IEDriverServer.exe
+REG ADD "HKCU\SOFTWARE\Microsoft\Windows\CurrentVersion\Run" /V "Selenium Hub" /t REG_SZ /F /D "C:\selenium\selenium.bat"
+REG ADD "HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Internet Explorer\Main\FeatureControl\FEATURE_BFCACHE"
+REG ADD "HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Internet Explorer\Main\FeatureControl\FEATURE_BFCACHE" /v iexplore.exe /t REG_DWORD /d 0 /f
+REG ADD HKLM\SOFTWARE\Microsoft\Windows\CurrentVersion\Policies\System /v EnableLUA /t REG_DWORD /d 0 /f
+timeout 5
+shutdown -s -t 0 -f
+EOF
+
+CR=$(printf '\r')
+sed -i "s/\$/$CR/" *.bat Autorun.inf
+genisoimage -output wininit.iso -volid cidata -joliet *.bat *.exe selenium.jar Autorun.inf
+
+echo "Converting vmdk to qcow2..."
+qemu-img convert -f vmdk -O qcow2 *.vmdk $ORIGIN
+echo "MANUAL part of installation, open graphics console and run D:\\install.bat"
+virt-install -n $VM_NAME -r 4000 --vcpus=4 --os-type=windows --disk $ORIGIN,device=disk --network user -c wininit.iso --noautoconsole --wait=-1 --noreboot
+echo "Installation finished"
+# virsh destroy $VM_NAME
+# virsh undefine $VM_NAME
```

Comment:
I don't think so. At the current time, there's enough back and forth iteration on this every 30 days, and a human already needs to be involved. I left them commented intentionally.

As it currently stands there's exactly one manual command to be run inside the VM. However after 30 days that may change. I want to make this as painless as possible, hence leaving these commands there.